tests/unit/models/test_model_interfaces.py

```diff
@@ -989,7 +1105,8 @@ def test_gaussian_process_deep_copyable(gpr_interface_factory) -> None:
 
     # check that updating the original doesn't break or change the deepcopy
     x_new = tf.concat([x, tf.constant([[10.0], [11.0]], dtype=gpflow.default_float())], 0)
-    new_data = Dataset(x_new, _3x_plus_10(x_new))
+    new_data = Dataset(x_new, _2sin_x_over_3(x_new))
+    print(new_data)
```

Comment:
print is not needed here, right?

Reply:
Oh yep you're right, I put that in while debugging and forgot to remove it!